src/main/handlers/status_check.py

```diff
@@ -38,7 +38,7 @@ def get_status(self):
                 self.maintenance = True
                 return discord.Status.dnd, discord.Game('maintenance')
             else:
-                game_str = f'{data["map"]} | {data["players"]} player' + ('s' if data["players"] != 1 else '')
+                game_str = f'{data["players"]} player' + ('s' if data["players"] != 1 else '') + f' | {data["map"]}'
```

Comment:
```suggestion
                game_str = f'{data["players"]} player{"s" if data["players"] != 1 else ""} | {data["map"]}'
```
could be done without string concatenation